### ISSUE TITLE
Screen Scaling on High DPI problem...

### ISSUE THREAD

**Issue by IPeluchito** [stella-emu/stella#700]
I just discovered that the emulator has a problem with high resolution screen scaling in **Windows 8.1** at **3840 x 2160** and **250% scaling** ... I leave screenshot comparing how it looks by default **(LEFT)** and disabling screen scaling on high DPI settings **(RIGHT)**

![86983966-e6b01580-c152-11ea-8123-d9b7e404781d](https://user-images.githubusercontent.com/64821708/92669440-29b77200-f2d7-11ea-9003-e29a2152b9f4.png)

**Comment by thrust26** [comment]
@DirtyHairy @sa666666 Can you verify this? I have no such display.

@IPeluchito If I get you right the scaling is done by the OS (Windows). What do you expect from Stella here? AFAIK Stella cannot prevent Windows from scaling. Or am I missing something?

Also, is the screenshot on the left done by Windows (ALT+Print) or by Stella (F12)?

**Comment by DirtyHairy** [comment]
Scaling on MacOS is crisp as Stella runs in "retina-native" mode there. I have no idea whether an equivalent exists on windows or how to use it.

**Comment by SpiceWare** [comment]
At work I added HiDPI support back in 2012.  I don't recall specifics, but I had to add a manifest file as specified here:
https://docs.microsoft.com/en-us/windows/win32/direct2d/how-to--size-a-window-properly-for-high-dpi-displays

Note: that's from 2018 so not where I originally found the info, but the manifest matches

In 2017 I updated the manifest because we ran into an issue with Windows 10 reporting X scaling values differently than the prior versions of Windows. Basically earlier versions of Windows used 133% scaling for X if you had selected 125%. 

The zip contains the manifest with Windows 10 updates, a screenshot of were to add the manifest, and adjust functions that differentiate between Windows 10 and prior.

[DPI.zip](https://github.com/stella-emu/stella/files/5328481/DPI.zip)
